blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_8_detailing_of_reinforcement_and_prestressing_tendons/formula_8_2.py

```diff
@@ -135,7 +135,7 @@ def latex(self) -> LatexFormula:
         return LatexFormula(
             return_symbol=r"\eta_2",
             result=f"{self:.2f}",
-            equation=r"\begin{matrix} 1.0 & \text{for }Ø ≤ 32 \\ (132 - Ø) / 100 & \text{for }Ø > 32  \end{matrix}",
+            equation=r"\begin{matrix} 1.0 & \text{for }\Phi ≤ 32 \\ (132 - \Phi) / 100 & \text{for }\Phi > 32  \end{matrix}",
```

Comment:
```suggestion
            equation=r"\begin{matrix} 1.0 & \text{for }Ø ≤ 32 \\ (132 - Ø) / 100 & \text{for }Ø > 32  \end{matrix}",
```

Reply:
Thank you for the feedback. the \Phi is indeed not perfect, lets change it back! Ill resolve the other issues later this week.